color-factory.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./formats/utils/color-factory/color-factory');
```

Comment:
Will we need to add type definitions for this in a `d.ts` file to consume this in TS projects without casting as `any`?

Reply:
Effort for adding a manually created typing file is low so let's add it